src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="productcard" data-qa="card">
+      <div class="images">
+        <img
+          class="images_item"
+          src="images/imac.jpeg"
+         alt="foto"
+     >
+      </div>
+      <div class="description">
+        <div class="description__name">
+        APPLE A1419 iMac 27" Retina 5K
+        Monoblock (MNED2UA/A)
+        </div>
+        <div class="description_code">
+          Product code: 195434
+        </div>
+        <div class="description__rating">
+          <div class="stars">
```

Comment:
there is no stars--4 modificator, check task checklist

Reply:
is not fixed. 
Suggest to be `stars stars--4` not `star stars--4`